lib/cli/src/generators/VUE3/index.ts

```diff
@@ -0,0 +1,9 @@
+import { baseGenerator, Generator } from '../baseGenerator';
+
+const generator: Generator = async (packageManager, npmOptions, options) => {
+  baseGenerator(packageManager, npmOptions, options, 'vue3', {
+    extraPackages: ['vue-loader@^16.0.0'],
```

Comment:
This is now e2e testing the version string specifying on `extraPackages` added in #13774